woocommerce/woocommerce-gutenberg-products-block
# Add Storybook documentation and remove deprecated knobs

Now that the Storybook [migration of existing stories](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5248) is complete, we can remove the knobs dependencies as they have been fully replaced by controls.

This PR is mostly just removing unused things, and adding a `README` file and VSCode snippets for smoother Storybook development.

However, while removing the knobs, I have noticed another story that hadn't been migrated because it was not noticed due to an error which didn't make it appear in the list of available stories. For now, I quickly migrated them to the new controls, but didn't really polish them: they all render but they are not as refined as the other ones in the epic, so I have added `todo` comments to make sure we follow-up and make it clear about their status. 

### Testing

### Manual Testing

Not really much testing to do for this PR. Perhaps just check the docs make some sense.
